Sources/MapboxNavigation/RouteMapViewController.swift

```diff
@@ -405,7 +405,9 @@ class RouteMapViewController: UIViewController {
     }
     
     @objc func resetFrameRate(_ sender: UIGestureRecognizer) {
-        navigationMapView.mapView.preferredFPS = NavigationMapView.FrameIntervalOptions.defaultFramesPerSecond
+        mapView.update {
```

Comment:
Will it be a problem to make this call asynchronous? If I’m not mistaken, the purpose of this code is to ensure that pans and other gestures trigger a smooth frame rate to avoid a stuttering effect. Hopefully there’s no delay built into this call that would cause the gesture to stutter for a moment.

Reply:
Doesn't seem like there is no issue in this particular case.